pandas/core/arrays/integer.py

```diff
@@ -187,6 +187,9 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
             raise TypeError("{} cannot be converted to an IntegerDtype".format(
                 values.dtype))
 
+    elif is_bool_dtype(values) and (is_integer_dtype(dtype) or dtype is None):
```

Comment:
can you add a 1-line comment here on acceptable dtypes

Reply:
```suggestion
   # convert boolean to int if target dtype is integer or unspecified
    elif is_bool_dtype(values) and (is_integer_dtype(dtype) or dtype is None):
```
Ok?

Reply:
more about the reason you need to check, and when/how dtype is None

Reply:
How about this? Couldn't fit it into one line, though.
```suggestion
    if the array is boolean, we first want to convert it to int to continue execution,
    if the target dtype is unspecified Int64 would be used later.
    elif is_bool_dtype(values) and (is_integer_dtype(dtype) or dtype is None):
```